.travis.yml

```diff
@@ -24,15 +24,15 @@ env:
     - COVERALLS=false
     - NOSE_FLAGS="--processes=2 --process-timeout=400 --no-open-files --with-timer --timer-top-n 50"
     - NOSE_TEST_LIST="analysis auxiliary coordinates core formats topology utils"
-    - PYTEST_FLAGS="--disable-pytest-warnings"
+    - PYTEST_FLAGS="--disable-pytest-warnings -n 2"
```

Comment:
can you please use `--numprocesses` it's more explicit and I'm trying to get upstream ci-helpers to detect that to auto install pytest-xdist.